packages/react-components/src/utils/story.ts

```diff
@@ -0,0 +1,11 @@
+import { DecoratorFunction, ArgTypes } from '@storybook/addons';
+
+export interface CSFStory<StoryFnReturnType = unknown> {
+  story?: {
+    name?: string;
+    decorators?: DecoratorFunction<StoryFnReturnType>[];
+    parameters?: { [name: string]: unknown };
+  };
+  argTypes?: ArgTypes;
+  (): StoryFnReturnType;
+};
```

Comment:
Currently `tsbuildconfig.json` `rootDir` is `src/lib`, utils dir is a sibling of the `src\lib` folder, `lerna bootstrap` fails

Reply:
@syncush utlils moved under lib